sdk/communication/azure-communication-email/README.md

```diff
@@ -0,0 +1,181 @@
+# Azure Communication Email client library for Python
+
+This package contains a Python SDK for Azure Communication Services for Email.
+
+## Key concepts
+
+The Azure Communication Email package is used to do following:
+- Send emails to multiple types of recipients
+- Query the status of a sent email message
+
+## Getting started
+
+### Prerequisites
+
+You need an [Azure subscription][azure_sub], a [Communication Service Resource][communication_resource_docs], and an [Email Communication Resource][email_resource_docs] with an active [Domain][domain_overview].
+
+To create these resource, you can use the [Azure Portal][communication_resource_create_portal], the [Azure PowerShell][communication_resource_create_power_shell], or the [.NET management client library][communication_resource_create_net].
+
+### Installing
+
+Install the Azure Communication Email client library for Python with [pip](https://pypi.org/project/pip/):
+
+```bash
+pip install azure-communication-email
+```
+
+## Examples
+
+`EmailClient` provides the functionality to send email messages .
+
+## Authentication
```

Comment:
Could we please add a sample on how I would authenticate using azure-identity?
It's usually how we would recommend customers authenticate.